src/conversions.jl

```diff
@@ -591,6 +591,43 @@ function cal2jd(iy, im, id)
 end
 
 
+@inline function d2tf(ndp, days)
+    sign = (days >= 0.0 ) ? '+' : '-'
+    a = SECONDS_PER_DAY * abs(days)
+
+    if ndp < 0
```

Comment:
I do not know whether we all these complications. The ERFA code says this about the `ndp` parameter:

> The largest positive useful value for ndp is determined by the
> size of days, the format of double on the target platform, and
> the risk of overflowing ihmsf[3].  On a typical platform, for
> days up to 1.0, the available floating-point precision might
> correspond to ndp=12.  However, the practical limit is typically
> ndp=9, set by the capacity of a 32-bit int, or ndp=4 if int is
> only 16 bits.

In Julia floating point precision is not platform-dependent so only the overflow risk remains. What do you think @giordano?

Reply:
To be honest, it's not clear to me what `ndp` is used inside the function and how it's usually computed (couldn't find any function actually using this one).  The comment you quoted makes me think that `ndp` could be somehow related to `eps(typeof(days))`?